.github/workflows/wheels-manylinux-build.yml

```diff
@@ -115,14 +115,14 @@ jobs:
           set -eo pipefail
 
           export MATRIX="
-          - { ctk: '11.8.0', arch: 'amd64', python: '3.9', image: 'rapidsai/manylinux_v2_2014:cuda-devel-11.8.0-centos7'}
-          - { ctk: '11.8.0', arch: 'amd64', python: '3.10', image: 'rapidsai/manylinux_v2_2014:cuda-devel-11.8.0-centos7' }
-          - { ctk: '11.8.0', arch: 'arm64', python: '3.9', image: 'rapidsai/manylinux_v2_2_31:cuda-devel-11.8.0-ubuntu20.04' }
-          - { ctk: '11.8.0', arch: 'arm64', python: '3.10', image: 'rapidsai/manylinux_v2_2_31:cuda-devel-11.8.0-ubuntu20.04' }
-          - { ctk: '12.0.1', arch: 'amd64', python: '3.9', image: 'rapidsai/manylinux_v2_2014:cuda-devel-12.0.1-centos7'}
-          - { ctk: '12.0.1', arch: 'amd64', python: '3.10', image: 'rapidsai/manylinux_v2_2014:cuda-devel-12.0.1-centos7' }
-          - { ctk: '12.0.1', arch: 'arm64', python: '3.9', image: 'rapidsai/manylinux_v2_2_31:cuda-devel-12.0.1-ubuntu20.04' }
-          - { ctk: '12.0.1', arch: 'arm64', python: '3.10', image: 'rapidsai/manylinux_v2_2_31:cuda-devel-12.0.1-ubuntu20.04' }
+          - { ctk: '11.8.0', arch: 'amd64', python: '3.9', linux_ver: 'centos7' }
```

Comment:
I noticed that these matrix variables are different from the corresponding conda jobs. Not something that we need to deal with now, but maybe worth unifying at some later point once we've merged the two container repos and moved more logic to repo-local scripts so that we can more fully align the wheel and conda workflows.